navigation_onboarding/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include "main.hpp"
+
+bool doNavigation(){
```

Comment:
This is one of many formatting errors. I would not advise you to include these as formatting should not be thought of as a manual thing to do. Instead, adapt CMake to automatically format (or add a git hook that checks formatting and rejects any faulty files; this is probably better).

Reply:
Also, why does this function return a bool? It's not clear to me what this is supposed to be, based on reading the signature.

Reply:
Supposed to return true if we need to brake (mostly just a placeholder function to give an idea of what navigation may look like in context of the run as a whole). Probably badly designed but not supposed to be used for any tasks during onboarding.

Reply:
The problem is that it's hard to infer that from the name. One thing that you could do is create an enum class that has values `kBrake` and `kAccelerate` which is returned by this function. You probably still want to change the name, though.